contracts/Libraries/Tupler.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity ^0.8.4;
+
+contract LibTupler {
+    function extractElement(bytes memory tuple, uint256 index)
+        public
+        pure
+        returns (bytes memory)
```

Comment:
Can we just make this return `bytes32` instead?

Reply:
ah sure, makes more since since this has fixed output size